CHANGES.rst

```diff
@@ -49,6 +49,9 @@ astropy.io.votable
 astropy.modeling
 ^^^^^^^^^^^^^^^^
 
+- Major rework of modeling internals. See release notes for details. Eliminates
```

Comment:
we might want to  have different wording in the new feature section compared to the API change section, but either way I think this is a very big change to list at both section.